src/publish-scoped.js

```diff
@@ -60,6 +64,37 @@ function updateLockFiles(packageName) {
   updateLockFileIfExists('package-lock.json', packageName);
 }
 
+function fetchJSONSync(url) {
+  const result = spawnSync('curl', ['--silent', url], {encoding: 'utf8'}).stdout;
+  return JSON.parse(result);
+}
+
+function verifyWixPackage(packageName) {
+  try {
+    const packageContents = fetchJSONSync(`http://npm.dev.wixpress.com/${packageName}/latest`);
+    const result = packageContents.publishConfig && packageContents.publishConfig.registry;
+    return Boolean(
+      result && (
+        result.includes('npm.dev.wixpress.com') ||
+        result.match(/https?:\/\/repo.dev.wix\//)
+      )
+    );
+  } catch (e) {
+    console.error(`An error occured while looking for ${packageName} in Wix's registry:`, e);
```

Comment:
Just a thought: maybe you should rethrow error and stop the flow?
Cause now if connection issue will appear with artifactory, you will treat as package doesn't exist.
Though a build will succeed and we will have mismatched versions in artifactory and developer will not even know about this.

Reply:
The only problem I have with this flow is that I'd rather cancel the whole flow if that's the case. wdyt?